docker-compose.yml

```diff
@@ -10,11 +10,11 @@ services:
     depends_on:
       - snapshot-mysql
     ports:
-      - "8080:8080"
+      - "3001:3001"
 
   # Snapshot MySQL instance
-  snapshot-mysql:
-    image: mysql:8.0.21
+  snapshot-mysql:  
+    image: mariadb:10.6
```

Comment:
Why using mariadb here?

Reply:
It could be my mistake but basically I was unable to make it run with mysql: I am unsure if I'm the only one with this issue: I tried 2 different times from scratch and I was never able to even reach the db.
when switching to mariadb it actually worked instantly so i pushed that too 

(obviously if you use this configuration in the production env then I probably wouldn't advise to merge this pr)

---
 
When running `docker-compose build` + up I met several issues
- unble to access mysql from the host machine: this disallows the use of any tool to interact with the db and, more importantly,  import the initial schema (also made me wonder if the db was really up or not) 
- in console this further error appeared;
```
code: 'ER_NOT_SUPPORTED_AUTH_MODE',
snapshot-hub      |   errno: 1251,
snapshot-hub      |   sqlMessage: 'Client does not support authentication protocol requested by server; consider upgrading MySQL client',```

Reply:
Which version of MySQL were you using?

Reply:
@stormsson Does this maybe solve your mysql issues: https://stackoverflow.com/a/50377944/2560696